README.md

```diff
@@ -1,141 +1,481 @@
+
+- [ODBC Troubleshooting: How to Enable Driver-manager Tracing](https://www.simba.com/blog/odbc-troubleshooting-tracing/)
+- [How do I turn unixODBC tracing on or off?](https://www.easysoft.com/support/kb/kb00945.html)
+
+### Enabling driver manager tracing: iODBC
+
+Comprehensive explanations (possibly, with some irrelevant vendor-specific details though) on how to enable ODBC driver manager tracing could be found at the following links:
+
+- [ODBC Troubleshooting: How to Enable Driver-manager Tracing](https://www.simba.com/blog/odbc-troubleshooting-tracing/)
+- [Tracing Application Behavior](http://www.iodbc.org/dataspace/doc/iodbc/wiki/iodbcWiki/FAQ#Tracing%20Application%20Behavior)
+
+### Building from sources: Windows
+
+#### Build-time dependencies <!-- omit in toc -->
+
+CMake bundled with the recent versions of Visual Studio can be used.
+
+An SDK required for building the ODBC driver is included in Windows SDK, which in its turn is also bundled with Visual Studio.
+
+You will need to install WiX toolset to be able to generate `.msi` packages. You can download and install it from [WiX toolset home page](https://wixtoolset.org/).
+
+#### Build steps <!-- omit in toc -->
+
+All of the following commands have to be issued in Visual Studio Command Prompt:
+
+- use `x86 Native Tools Command Prompt for VS 2019` or equivalent for 32-bit builds
+- use `x64 Native Tools Command Prompt for VS 2019` or equivalent for 64-bit builds
+
+Clone the repo with submodules:
+
+```sh
+git clone --recursive git@github.com:ClickHouse/clickhouse-odbc.git
+```
+
+Enter the cloned source tree, create a temporary build folder, and generate the solution and project files in it:
+
+```sh
+cd clickhouse-odbc
+mkdir build
+cd build
+
+# Configuration options for the project can be specified in the next command in a form of '-Dopt=val'
+
+# Use the following command for 32-bit build only.
+cmake -A Win32 -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
+
+# Use the following command for 64-bit build only.
+cmake -A x64 -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
+```
+
+Build the generated solution in-place:
+
+```sh
+cmake --build . -C RelWithDebInfo
+cmake --build . -C RelWithDebInfo --target package
+```
+
+...and, optionally, run tests (note, that for non-unit tests, preconfigured driver and DSN entries must exist, that point to the binaries generated in this build folder):
+
+```sh
+cmake --build . -C RelWithDebInfo --target test
+```
+
+...or open the IDE and build `all`, `package`, and `test` targets manually from there:
+
+```sh
+cmake --open .
+```
+
+### Building from sources: macOS
+
+#### Build-time dependencies <!-- omit in toc -->
+
+You will need Xcode 10 or later and Command Line Tools to be installed, as well as [Homebrew](https://brew.sh/).
+
+#### Build-time dependencies: iODBC <!-- omit in toc -->
+
+Homebrew: execute the following in the terminal (assuming you have [Homebrew](https://brew.sh/) installed):
+
+```sh
+brew install git cmake make poco openssl libiodbc
```

Comment:
poco?

Reply:
yes, for cases when the driver is built with unboundled poco.